Source/Csla/BusinessBindingListBase.cs

```diff
@@ -812,8 +812,10 @@ public virtual bool IsSavable
     {
       get
       {
-        bool auth = Rules.BusinessRules.HasPermission(ApplicationContext, Rules.AuthorizationActions.EditObject, this);
-        return (IsDirty && IsValid && auth && !IsBusy);
+        if (!IsDirty || !IsValid || IsBusy)
```

Comment:
Style wise I generally prefer to have one `return` in a method. I find it easier to read.

Reply:
To adhere to that I have two options:
1. Change the code to: `return IsDirty && IsValid && !IsBusy && Rules.BusinessRules.HasPermission(ApplicationContext, Rules.AuthorizationActions.EditObject, this);`
2. Remove the change in this file because I can't use a temporary variable here. The goal is to avoid the `HasPermission()` call when not necessary.

Reply:
What about 

```csharp
  bool result = (IsDirty && IsValid && !IsBusy);
  if (result)
  {
    // check authz here
  }
  return result;
```

Reply:
Done.